src/wasm-linker.h

```diff
@@ -132,14 +90,16 @@ class Linker {
     assert(implementedFunctions.count(name));
   }
 
-  // Allocate the user stack, set up the initial memory size of the module, lay
-  // out the linear memory, process the relocations, and set up the indirect
-  // function table.
-  void layout();
+  bool isEmpty() {
```

Comment:
I'm still not happy about this. In theory it should call a Module.isEmpty() utility. But even that seems odd...

Is this actually checking if the module is empty? Or whether it has already been written to?